examples/recipegenerator/recipegenerator.gpt

```diff
@@ -0,0 +1,17 @@
+tools: sys.find, sys.read, sys.write, recipegenerator, tool.gpt
+
+Perform the following steps in order:
+
+1. Give me a list of 5 to 10 ingredients from the picture grocery.png located in the current directory and put those extracted ingredients in a file name response.json
```

Comment:
Instead of writing to a file, can the list of ingredients be passed to recipe generator tool directly? Have you already tried this? 

You might need to update the tool description `Generate a recipe from an ingredient list`
args: ingredients: a comma separated list of available ingredients.

Reply:
I tweaked the steps and it worked. Removing that extra json file from this.